documentation/configuration.md

```diff
@@ -164,3 +174,17 @@ end
 
 To make modules available in the provider state set_up and tear_down blocks, include them in the configuration as shown below. One common use of this to include factory methods for setting up data so that the provider states file doesn't get too bloated.
 
+### interactions_replay_order
+
+Default value: `:recorder`
+Options: `:recorder`, `:random`
+
+Replays interactions in a specific order. In combination with pactfile_write_order will allow you to have a consistent pact contract replayed in random order.  
+
+```ruby
+Pact.configure do | config |
+  config.interactions_replay_order = :random
+  config.pactfile_write_order = :alphabetical
```

Comment:
`config.pactfile_write_order = :alphabetical` - This line will only exist in the consumer. interactions_replay_order and pactfile_write_order will never exist in the same project as one is for the consumer, and one for the provider.